graspologic/nominate/spectralVN.py

```diff
@@ -0,0 +1,290 @@
+    in order of relation to members of S.
+    Spectral Vertex Nomination solves this problem by embedding G
+    into a low dimensional euclidean space using any of the embedding
+    algorithms (:ref:`tutorials <embed_tutorials>`),
+    and then generating a nomination list by some distance based algorithm.
+    In the simplest case, for each seed vertex u, the other vertices are
+    ranked in order of euclidean distance from u.
+    There can be both attributed and unattributed cases. In the unattributed
+    case, we treat each seed vertex individually and rank all vertices by distance.
+    In the attributed case, subsets of the seed vertices share some attribute we
+    care about. We want to rank the other vertices by the likelihood that they
+    also share that attribute. Note that the unattributed problem is simply the
+    attributed problem when each seed vertex has a unique attribute.
+    SVN algorithms in general make the assumption that vertexes
+    are likely to be related in a way that is of interest if they are close to
+    each other in an embedding. This is a somewhat simplified notion of relatedness,
+    and is not appropriate for all Vertex Nomination problems.
+
+    Parameters
+    ----------
+    embedding: np.ndarray, default = None
+        A pre-calculated embedding may be provided, in which case
+        it will be used for vertex nomination instead of embedding
+        the adjacency matrix using embeder.
+    embeder: str or BaseEmbed, default = 'ASE'
+        May provide either a embed object or a string indicating
+        which embedding method to use, which may be either
+        "ASE" for :py:class:`~graspologic.embed.AdjacencySpectralEmbed` or
+        "LSE" for :py:class:`~graspologic.embed.LaplacianSpectralEmbed`.
+    persistent : bool, default = True
+        If ``False``, future calls to fit will overwrite an existing embedding. Must be ``True``
+        if an embedding is provided.
+
+
+    Attributes
+    ----------
+    embedding : np.ndarray
+        The spectral embedding of the graph that spectral nomination will be preformed on.
+    embeder : :py:class:`~graspologic.embed.BaseEmbed`
+        The embed object to be used to compute the embedding.
+    attr_labels : np.ndarray
+        The attributes of the vertices in the seed (parameter 'y' for fit).
+        Shape is the number of seed vertices.
+    unique_att : np.ndarray
+        Each unique attribute represented in the seed. One dimensional.
+    distance_matrix : np.ndarray
+        The euclidean distance from each seed vertex to each vertex.
+        Shape is ``(number_vertices, number_unique_attributes)`` if attributed
+        or Shape is ``(number_vertices, number_seed_vertices)`` if unattributed.
+    persistent : bool
+        If ``False``, future calls to fit will overwrite an existing embedding. Must be ``True``
+        if an embedding is provided.
+
+    References
```

Comment:
These references should be numbered, and indicate in the text where they are referenced